Handle spaces in source directory Path

p-galligan opened the pull request:
Handles spaces in original file and derivative paths.

Updates readme.

Fixes #67

Commit:
handle spaces in directories

Handles spaces in original file and derivative paths.

Updates readme

helrond commented on iiif_pipeline/derivatives.py:
```diff
@@ -65,11 +65,13 @@ def create_jp2(files, identifier, derivative_dir, replace=False):
+                       "-o", derivative_path,
+                       "-n", str(layers),
+                       "-SOP"] + default_options
+                print(cmd)
```
Stray print

helrond reviewed (approved):
Can you add a test for this?

helrond reviewed (changes requested):
Err I meant to request changes...

Commit:
add test iteration

Adds new source folders and derivative folders with spaces in them and then iterates over those folders in tests.
